warehouse/models/intermediate/gtfs_quality/guidelines_checks/int_gtfs_quality__schedule_download_success.sql

```diff
@@ -10,27 +12,29 @@ fct_schedule_feed_downloads AS (
 
 daily_feed_download_unzip_success AS (
     SELECT date,
-           base64_url
+           base64_url,
+           LOGICAL_OR(download_success) AS download_success,
+           LOGICAL_OR(unzip_success) AS unzip_success
       FROM fct_schedule_feed_downloads
-     WHERE download_success
-       AND unzip_success
+      GROUP BY 1, 2
 ),
 
-int_gtfs_quality__schedule_download_success AS (
+int_gtfs_quality__schedule_url_download_success AS (
     SELECT
-        idx.date,
-        idx.base64_url,
-        {{ static_feed_downloaded_successfully() }} AS check,
-        {{ compliance_schedule() }} AS feature,
+        idx.* EXCEPT(status),
         CASE
-            WHEN LOGICAL_AND(d.base64_url IS NOT null) THEN {{ guidelines_pass_status() }}
-            ELSE {{ guidelines_fail_status() }}
+            WHEN download_success AND unzip_success THEN {{ guidelines_pass_status() }}
+            -- means we attempted to download and failed
+            WHEN idx.has_schedule_url AND d.base64_url IS NOT NULL THEN {{ guidelines_fail_status() }}
```

Comment:
is the idea with the fail condition that there is basically an implied "NOT(download_success AND unzip_success)"?

Reply:
Yeah, the order of evaluation matters, so if it wasn't caught in the first condition but we have a schedule URL on the right side that means that we did attempt download and got some other outcome

Reply:
yep makes sense! I probably would have made that explicit but your way's good